js/controller.js

```diff
@@ -420,7 +420,7 @@ OO.plugin("Html5Skin", function (OO, _, $, W) {
 
       this.state.upNextInfo.timeToShow = timeToShow;
 
-      if (duration - currentPlayhead <= timeToShow &&
+if (duration!==0 &&- currentPlayhead <= timeToShow &&
```

Comment:
yeah it has to be ,  if (duration !== 0 && duration - currentPlayhead <= timeToShow &&

will fix it.